.env.template

```diff
@@ -1,14 +1,8 @@
-ENABLE_PLUGIN=
-ENABLE_TEST_PROPERTIES=
-ENABLE_BC_PROVIDER=
 CHANGELOG_URL=
 ENABLE_PRELOADING=
-ENABLE_NEW_SETTING_MODAL=
-ENABLE_SQLITE_PROVIDER=
 ENABLE_NEW_SETTING_UNSTABLE_API=
-ENABLE_NOTIFICATION_CENTER=
-ENABLE_CLOUD=
-ENABLE_MOVE_DATABASE=
-SHOULD_REPORT_TRACE=
-TRACE_REPORT_ENDPOINT=
-CAPTCHA_SITE_KEY=
\ No newline at end of file
+ENABLE_CAPTCHA=
+CAPTCHA_SITE_KEY=
+ENABLE_ENHANCE_SHARE_MODE=
+ALLOW_LOCAL_WORKSPACE=
+DEBUG_JOTAI=
```

Comment:
what is this?

Reply:
inject jotai devtools. it was used but this flag was never set. I add it in case of some one need